Art/thiunuwan/styles.css

```diff
@@ -0,0 +1,30 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  margin: 0;
+  background-color: #121212;
+}
+
+.hero__image {
+  outline: 4px solid #1ed9b7;
+  /* background-image: url('img/me.png'); */
+  display: block;
+  width: 350px;
+  height: auto;
+  /*box-shadow: inset 0 0 0 9px rgb(203, 37, 37);*/
```

Comment:
# Feedbacks
- [ ] Unnecessary comments
Please remove the unnecessaries comments on l:12 and l:16
# Context
```css
/* background-image: url('img/me.png'); 


/*box-shadow: inset 0 0 0 9px rgb(203, 37, 37);*/

```